admin/Dockerfile.qa

```
@@ -57,6 +58,18 @@ ENV \
 ENV \
   XDG_CACHE_HOME=$NOX_ENV_DIR/.cache
 
+RUN git clone \
+    -n \
+    -q https://github.com/google/googletest.git /tmp/googletest \
+  && cd /tmp/googletest \
+  && git checkout 58d77fa8070e8cec2dc1ed015d66b454c8d78850 \
```

Can use a shallow clone if we refer to the target commit by its tag name:

```suggestion
RUN git clone -q \
    --branch=release-1.12.1 \
    --depth=1 \
    https://github.com/google/googletest.git /tmp/googletest \
  && cd /tmp/googletest \
```

---

The suggestion from CMake tutorial was to rely on the commit ID because of security reasons.
I think here we can switch to the shallow clone.